manifests/init.pp

```diff
@@ -74,11 +74,17 @@
   String[1] $root_store            = 'LocalMachine',
   String[1] $store_dir             = 'My',
   Stdlib::Windowspath $scripts_dir = 'C:\temp',
-  Boolean $exportable              = true
+  Boolean $exportable              = true,
+  Optional[Boolean] $wildcard      = false,
```

Comment:
Does `Optional[]` make sense here? It's preferred to always initiate a Boolean with `true` or `false` and not allowing `undef`.